.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  pull_request: {}
+  push: {}
+
+name: Continuous Integration
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [12, 14, 16, 18]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node_version }}
+          cache: 'npm'
+          cache-dependency-path: '**/package.json'
```

Comment:
Ideally you wouldn’t specify `cache-dependency-path`, but by default it uses the lockfile and this repo doesn’t have a lockfile. It might be a good idea to commit one (it doesn’t get used if the package is installed by someone else as a dependency, unlike *shrinkwrap* files, so it’s OK for libraries like this) since it lets you get reliable about which versions of dev dependencies you use in CI, but not necessarily required.